doordeck-sdk/src/commonMain/kotlin/com/doordeck/multiplatform/sdk/internal/api/FusionClient.kt

```diff
@@ -8,18 +8,22 @@ import com.doordeck.multiplatform.sdk.api.responses.DoorStateResponse
 import com.doordeck.multiplatform.sdk.api.responses.FusionLoginResponse
 import com.doordeck.multiplatform.sdk.api.responses.IntegrationConfigurationResponse
 import com.doordeck.multiplatform.sdk.api.responses.IntegrationTypeResponse
+import com.doordeck.multiplatform.sdk.internal.ContextManagerImpl
 import com.doordeck.multiplatform.sdk.util.addRequestHeaders
 import io.ktor.client.HttpClient
 import io.ktor.client.request.setBody
 
 internal open class FusionClient(
-    private val httpClient: HttpClient
+    private val httpClient: HttpClient,
+    private val contextManager: ContextManagerImpl
 ) : AbstractResourceImpl() {
 
     suspend fun loginRequest(email: String, password: String): FusionLoginResponse {
```

Comment:
Do we need @DoordeckOnly on  the fusion APIs as well? I expect its in the parent but I haven't checked

Reply:
Added, it was missing!